src/has-permissions.in-jwt-token.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, it } from "vitest";
+
+import { hasPermissionsInJwtToken } from "./has-permissions-in-jwt-token";
+import { DashboardTokenPayload } from "./verify-jwt";
+
+describe("hasPermissionsInJwtToken", () => {
+  it("Pass if no required permissions, and user has none", async () => {
+    const tokenData: Pick<DashboardTokenPayload, "user_permissions"> = {
+      user_permissions: [],
+    };
+    await expect(hasPermissionsInJwtToken(tokenData)).toBeTruthy();
```

Comment:
avoid `toBeTruthy();` (maybe we can add eslint rule) -> toBeTruthy is doint type coercion so eg `!![]` will pass. This an lead to some unexpected false positives.

There is `toBe(true)` which is safe